test/functional/auth_source_ldaps_controller_test.rb

```diff
@@ -1,15 +1,11 @@
 require 'test_helper'
+require 'functional/shared/basic_rest_response_test'
 
 class AuthSourceLdapsControllerTest < ActionController::TestCase
-  def test_index
-    get :index, {}, set_session_user
-    assert_template 'index'
-  end
-
-  def test_new
-    get :new, {}, set_session_user
-    assert_template 'new'
-  end
+  include ::BasicRestResponseTest
```

Comment:
It might be easier to include this straight into the TestCase class in test_helper, or at least require the file from test_helper as it's used so much.